examples/illustrations/cell_site/drafting.ttl

```diff
@@ -0,0 +1,279 @@
+@prefix core: <https://ontology.unifiedcyberontology.org/uco/core/> .
+@prefix drafting: <http://example.org/ontology/drafting#> .
+@prefix observable: <https://ontology.unifiedcyberontology.org/uco/observable/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+drafting:AntennaFacet
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf core:Facet ;
+	rdfs:label "AntennaFacet"@en ;
+	rdfs:comment "An antenna alignment facet contains the metadata surrounding the cell tower's antenna position."@en ;
+	rdfs:seeAlso <https://case.atlassian.net/browse/ONT-469> ;
+	sh:property
+		[
+			sh:datatype xsd:decimal ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path drafting:azimuth ;
+		] ,
+		[
+			sh:datatype xsd:decimal ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path drafting:elevation ;
+		] ,
+		[
+			sh:datatype xsd:decimal ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path drafting:height ;
+		] ,
+		[
+			sh:datatype xsd:decimal ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path drafting:horizontalBeamWidth ;
+		] ,
+		[
+			sh:datatype xsd:decimal ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path drafting:signalStrength ;
+		] ,
+		[
+			sh:datatype xsd:decimal ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:nodeKind sh:Literal ;
+			sh:path drafting:skew ;
+		]
+		;
+	sh:targetClass observable:AntennaFacet ;
+	.
+
+drafting:CapturedTelecommunicationsInformation
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf observable:ObservableObject ;
+	rdfs:label "CapturedTelecommunicationsInformation"@en ;
+	rdfs:seeAlso <https://case.atlassian.net/browse/ONT-469> ;
+	sh:targetClass drafting:CapturedTelecommunicationsInformation ;
+	.
+
+drafting:CapturedTelecommunicationsInformationFacet
+	a
+		owl:Class ,
+		sh:NodeShape
+		;
+	rdfs:subClassOf core:Facet ;
+	rdfs:label "CapturedTelecommunicationsInformationFacet"@en ;
+	rdfs:comment "A captured telecommunications information facet represents certain information within captured or intercepted telecommunications data."@en ;
+	rdfs:seeAlso <https://case.atlassian.net/browse/ONT-469> ;
+	sh:property
+		[
+			sh:class drafting:CellSite ;
+			sh:maxCount "1"^^xsd:integer ;
+			sh:minCount "1"^^xsd:integer ;
+			sh:nodeKind sh:BlankNodeOrIRI ;
+			sh:path drafting:captureCellSite ;
```

Comment:
`captureCellSite` shares a similar issue with `partyDevice`, but I'm leaning to a different resolution that's part of a bigger question CASE needs to address as a community.

Say you had two `CapturedTelecommunicationsInformation` objects, and the ground truth is both are describing the same physical site.  But, say the two are part of different investigations that later become a joint investigation.  What are the values of the `captureCellSite` property for those two objects?  Do we need to instantiate an independent `CellSite` object for every CDR, recording the provenance of those graph nodes as being derived from the CDR?

I think the `captureCellSite` object encourages an investigation-independent knowledge base---and this is contrary to my recommendation to delete the "Party" device/subscription properties.  The party properties will refer to investigative subjects.  Cell sites more refer to ... terrain fixtures, for lack of a better phrase.

I've worked through a demonstration based on data from opencellid.org, now added to the README.  We can either:

1. Keep `captureCellSite` as an object reference.
2. Turn `captureCellSite` into a quartet of datatype properties.

@kchason , @cyberinvestigationexpress , and anyone else interested: please review the added example text about opencellid objects and reply with what you think is better.

Reply:
Lacking further input, we will keep `captureCellSite` an object reference.